src/Symfony/Bundle/FrameworkBundle/Resources/config/test.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+use Symfony\Bundle\FrameworkBundle\Test\TestContainer;
+use Symfony\Component\BrowserKit\CookieJar;
+use Symfony\Component\BrowserKit\History;
+use Symfony\Component\DependencyInjection\ServiceLocator;
+use Symfony\Component\HttpKernel\EventListener\TestSessionListener;
+
+return static function (ContainerConfigurator $container) {
+    $testServiceLocatorId = 'test.private_services_locator';
```

Comment:
Please don't use a variable for this. Otherwise we could start using variables for all the other service ids as well.

Reply:
Fixed!